janusgraph-core/pom.xml

```diff
@@ -116,15 +116,15 @@
         </testResources>
         <plugins>
             <plugin>
-                <groupId>org.codehaus.gmaven</groupId>
-                <artifactId>gmaven-plugin</artifactId>
+                <groupId>org.codehaus.gmavenplus</groupId>
+                <artifactId>gmavenplus-plugin</artifactId>
                 <version>1.5</version>
                 <executions>
                     <execution>
                         <goals>
                             <goal>generateStubs</goal>
                             <goal>compile</goal>
-                            <goal>generateTestStubs</goal>
+                            <goal>testGenerateStubs</goal>
```

Comment:
As far as testing goes, I was wary of the changes meaning that some files weren't generated correctly.  I tested by comparing the directory structures, including all target/generated folders, of a master repo vs one with these changes.